### ISSUE TITLE
RTG issues on Sidi 128 over HDMI

### ISSUE THREAD

**Issue by fvillanustre** [minimig-dev/MinimigAGA-MiST-TC64#14]
Hello everyone! I have encountered a problem with setting up RTG video modes for HDMI on Sidi128. I use 800x600x16 bits on VGA with no issue (Picasso96 2.0, latest minimig utils package, AmigaOS 3.2.2.1 and the corresponding kickstart rom) but that particular mode had some frequency synchronization issues on HDMI making it unusable on my monitor, so I decided to explore other modes. I found that 1440x900x8 works well, but I'd like a lower resolution mode with 16 bits, and this is where the problems begin. For all other modes that I tried (720p, 1024x768, etc.), there seems to be some sort of memory corruption on the video. The screen shows sometimes two or three large horizontal sections as if they were scanning memory used by other processes, rather than the video memory reserved by the minimig video driver (which, by default, reserves 4MB). As soon as this happens, the amiga environment freezes and there is no way to recover other than a reset (the OSD still works fine most of the time). Has anyone else seen the same problem?

1440x900x8 works fine, each screen takes roughly 1.3MB and I can open 2 screens at the same time without issues and switch among them very quickly.

Please let me know if there is something I'm doing wrong, or if you'd like more information that can help identify the root cause. 

Thanks!!!!

**Comment by robinsonb5** [comment]
Thanks for the report.  Could I ask you to test those lower resolutions with 8-bits, and let me know if the same thing happens?

**Comment by fvillanustre** [comment]
Yes, the same thing happens with 8 bits too. So far, the only mode that works reliably on HDMI for me is 1440x900x8. But on the same monitor, on VGA, I can use 1440x900x8 and also the 8 and 16 bits options for 800x600.
